apps/api/src/domain/models/audio-item/errors/transcript-line-item/cannot-add-inconsistent-line-item.error.ts

```diff
@@ -0,0 +1,15 @@
+import { InternalError } from '../../../../../lib/errors/InternalError';
+import { TranscriptItem } from '../../entities/transcript-item.entity';
+
+/**
+ * This is a top-level wrapper for any errors when adding a new line item to
```

Comment:
I feel like things are getting out of control with the depth of nested innerErrors.